en/notes-tidb-operator-v1.1.md

```diff
@@ -0,0 +1,138 @@
+
+### Monitor
+
+To create the TidbMonitor CR and manage the Monitor component, refer to [Monitor a TiDB Cluster using TidbMonitor](monitor-using-tidbmonitor.md).
+
+> **Note:**
+>
+> * The `metadata.name` in the TidbMonitor CR must be consistent with the name of the TidbCluster CR in the cluster.
+> * The way TiDB Operator renders resources is different from that of Helm. If you migrate configurations from `tidb-cluster chart values.yaml` to TidbMonitor CR, the Monitor component might be rolling updated.
+
+### Initializer
+
+- If the initialization job is executed before TiDB Operator is upgraded to v1.1, the initialization job does not need to be migrated from the tidb-cluster chart to the TidbInitializer CR.
+
+- If the initialization job is not executed before TiDB Operator is upgraded to v1.1, and the password for the TiDB root user is not modified, you need to initialize your cluster after upgrading to TiDB Operator v1.1. For details, refer to [Initialize a TiDB Cluster in Kubernetes](initialize-a-cluster.md).
+
+### Pump
+
+After TiDB Operator is upgraded to v1.1, you can modify the TidbCluster CR and add the configuration of Pump. You can then manage the Pump component using TidbCluster CR.
+
+``` yaml
+spec
+  ...
+  pump:
+    baseImage: pingcap/tidb-binlog
+    version: v3.0.11
+    replicas: 1
+    storageClassName: local-storage
+    requests:
+      storage: 30Gi
+    schedulerName: default-scheduler
+    config:
+      addr: 0.0.0.0:8250
+      gc: 7
+      heartbeat-interval: 2
+```
+
+You can modify `version`, `replicas`, `storageClassName`, `requests.storage`, and other configurations according to the needs of your cluster.
+
+> **Note:**
+>
+> The way TiDB Operator renders resources is different from that of Helm. If you migrate configurations from `tidb-cluster chart values.yaml` to TidbCluster CR, the Pump component might be rolling updated.
+
+### Scheduled Backup
+
+After TiDB Operator is upgraded to v1.1, you can configure the scheduled full backup using BackupSchedule CR:
+
+- If the TiDB cluster version < v3.1, refer to [Scheduled backup using mydumper](backup-to-s3.md#scheduled-full-backup-to-s3-compatible-storage)
+- If the TiDB cluster version >= v3.1, refer to [Scheduled backup using BR](backup-to-aws-s3-using-br.md#scheduled-full-backup-to-s3-compatible-storage)
+
+> **Note:**
+>
+> - Currently, with BackupSchedule CR, you can back up data only to S3 and GCS using Mydumper, and back up data to S3 using BR. If you perform the scheduled full backup and send data to local Persistent Volume Claim (PVC) before the upgrade, you cannot switch to the CR management after the upgrade.
+> - If you switch to the CR management, to avoid repeated backup, delete the Cronjob of the previous scheduled full backup.
+
+### Drainer
+
+- If Drainer is not deployed before TiDB Operator is upgraded to v1.1, you can deploy Drainer as in [Deploy multiple drainers](deploy-tidb-binlog.md#deploy-multiple-drainers).
```

Comment:
Is the [Deploy multiple drainers](deploy-tidb-binlog.md#deploy-multiple-drainers) link available? Please check.

Reply:
`maintain-tidb-binlog.md` is changed to `deploy-tidb-binlog.md` in #53(unmerged), so the CI check fails.